docs/index.rst

```diff
@@ -2,20 +2,21 @@ Welcome to the Certbot documentation!
 ==================================================
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 2
 
+   start
    intro
-   what
-   install
-   using
-   contributing
-   packaging
-   resources
```

Comment:
You need to add the file names of all the new .rst files you added here

Reply:
Question here: will adding them to this file be redundant with include directives? Should I remove  .. include:: what.rst from intro.rst and just put what in the index.rst file, indented twice as far?